### PR TITLE
add TabButton htmlable

### PR DESCRIPTION
**Pull request by patrickweh**
fix TicketCreatedNotificationListener.php

### CODE REVIEW

File: src/Console/Commands/Init/InitPermissions.php [hunk 1]
```diff
@@ -121,6 +125,23 @@ private function registerWidgetPermissions(): void
         }
     }
 
+    public function registerTabPermissions(): void
+    {
+        $this->info('Registering tab permissions');
+        $registry = app(ComponentRegistry::class);
+        foreach(invade($registry)->aliases as $alias => $component) {
+            if(! in_array(WithTabs::class, class_uses_recursive($component))) {
```
Comment (SirSplasch) [inline src/Console/Commands/Init/InitPermissions.php:133]: whitespace after if

File: src/Console/Commands/Init/InitPermissions.php [hunk 2]
```diff
@@ -121,6 +125,23 @@ private function registerWidgetPermissions(): void
...
+                continue;
+            }
```
Comment (SirSplasch) [inline src/Console/Commands/Init/InitPermissions.php:135]: new line after }

File: src/Console/Commands/Init/InitPermissions.php [hunk 3]
```diff
@@ -46,6 +49,7 @@ public function handle(): void
         $this->registerActionPermission();
         $this->registerRoutePermissions();
         $this->registerWidgetPermissions();
+        $this->registerTabPermissions();
 
         foreach ($this->currentPermissions as $id => $currentPermission) {
             Permission::query()->whereKey($id)->delete();
```
Comment (SirSplasch) [inline src/Console/Commands/Init/InitPermissions.php:55]: why not only one delete statement, which deletes all?

**Review by SirSplasch — CHANGES_REQUESTED** [review]

File: resources/views/livewire/order/order.blade.php [hunk 4]
```diff
@@ -88,17 +88,7 @@ class="w-full"
...
-                    'order.order-positions' => __('Order positions'),
-                    'order.attachments' => __('Attachments'),
-                    'order.accounting' => __('Accounting'),
-                    'order.comments' => __('Comments'),
-                    'order.related' => __('Related processes'),
-                    'order.activities' => __('Activities'),
-                ]"
-    >
+    <x-tabs wire:model.live="tab" :tabs="$tabs">
         <div class="w-full lg:col-start-1 xl:col-span-2 xl:flex xl:space-x-6">
```
[the remark this answers is not in the document]
Comment (patrickweh) [reply resources/views/livewire/order/order.blade.php:92]: yes, because the sections right and left persist for every tab

**Review by SirSplasch — APPROVED** [review]

**Pull request merged**